runtime/gc_modron_startup/mmparse.cpp

```diff
@@ -1472,6 +1472,19 @@ gcParseCommandLineAndInitializeWithValues(J9JavaVM *vm, IDATA *memoryParameters)
 	/* Parse the command line 
 	 * Order is important for parameters that match as substrings (-Xmrx/-Xmr)
 	 */
+	bool eanbleOriginalJDK8HeapSizeCompatibilityOption = false;
```

Comment:
This is pure C function, so all declarations must be done at the beginning of the function. This line does not create problem just because it happen still to be in declaration section. But in general the functionality (described in name) is very local to be defined in declaration section. The solution might be wrap this code to extra `{``}` to define local section.

Reply:
Declaration must be at the beginning of the function but my explanation why might be not entirely correct. This function uses `goto` and compiler does not allow jump over variable declaration

Reply:
The C rule that requires declarations at the beginning of the block (not necessarily the function) don't apply to a C++ source file even if the function is not associated with a class.